docs/source/caching.mdx

```diff
@@ -15,6 +19,63 @@ All caches used by the `ApolloClient` must conform to the [`NormalizedCache` pro
 
 All caches can be cleared in their entirety by calling [`clear(callbackQueue:completion:)`](api/Apollo/protocols/NormalizedCache/#clearcallbackqueuecompletion). If you need to work more directly with the cache, please see the [Direct Cache Access](#direct-cache-access) section.
 
+## Cache Setup
+
+### In-Memory Cache
+
+For `InMemoryNormalizedCache`, no sub-libraries are needed.
+
+This type of cache is used by default when setting up an `ApolloClient`. If you want to use an in-memory cache without modifications, all you have to do is instantiate an `ApolloClient` instance and not pass anything into the `store` parameter. 
+
+If for some reason you find you need to instantiate the in-memory cache yourself, you can do so with one line:
+
+```swift:title=Cache%20Setup
+import Apollo
+
+let cache = InMemoryNormalizedCache()
+```
+
+### SQLite Cache
+To use the `SQLiteNormalizedCache`, you must bring in the `ApolloSQLite` sub-library using your package manager of choice:
+
+<SPMSQLite />
+
+<CocoaPodsSQLite />
+
+<CarthageSQLite />
+
+You can then set up a file URL for your `SQLite` file, use that file URL to instantiate a SQLite cache, and then use that SQLite cache to instantiate an `ApolloStore`, which you then pass into `ApolloClient`'s initializer: 
+
+```swift:title=Client%20Setup
```

Comment:
Oh nice, does this %20 thing work? Need to keep this in mind 🤔

Reply:
Yep, found that hack a couple weeks ago: 
<img width="759" alt="Screen Shot 2020-07-14 at 6 43 12 PM" src="https://user-images.githubusercontent.com/1976498/87486887-eb1c7880-c601-11ea-815f-ef7710d923dd.png">